posthog/constants.py

```diff
@@ -304,6 +304,7 @@ class FlagRequestType(StrEnum):
 
 ENRICHED_DASHBOARD_INSIGHT_IDENTIFIER = "Feature Viewed"
 DATA_WAREHOUSE_TASK_QUEUE = "data-warehouse-task-queue"
+DATA_WAREHOUSE_TASK_QUEUE_V2 = "v2-data-warehouse-task-queue"
```

Comment:
You're going to need to register these workflows and the sync activities in the config in `posthog/management/commands/start_temporal_worker.py`

Reply:
will approve once these go in